### PR TITLE
Mark email sent by subscribing to a repo and Send new issue! as delivered #362

### PR DESCRIPTION
**Pull request by jacarandang**

### CODE REVIEW

File: app/models/repo_subscription.rb [hunk 1]
```diff
@@ -20,6 +20,7 @@ def self.for(repo_id)
     IssueAssigner.new(repo_sub.user, [repo_sub]).assign
     if assignment = repo_sub.user.issue_assignments.order(:created_at).eager_load(:repo_subscription)
                             .where(repo_subscriptions: { repo_id: repo_sub.repo_id }).last
+      assignment.update(delivered: true)
```
Comment (prathamesh-sonpatki) [inline app/models/repo_subscription.rb]: Let's use `update!` instead so that we know if it fails.

**Comment by prathamesh-sonpatki** [comment]
@jacarandang Can you add a test case for this?

**Comment by jacarandang** [comment]
For reference #362 
Will add tests tomorrow . Thanks

**Comment by prathamesh-sonpatki** [comment]
:heart:

**Pull request merged**